.changesets/use-log-level-config-option-instead-of-debug.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "fix"
+---
+
+Prefer the value of the `log_level` config option, instead of the deprecated `debug` config option, when deciding whether to log a debug message. If `log_level` does not have a valid value, the values of the deprecated `debug` and `transaction_debug_mode` config options are taken into account.
```

Comment:
```suggestion
Prefer the value of the `log_level` config option, instead of the deprecated `debug` config option, when deciding whether to log a debug message. If `log_level` does not have a (valid) value, the values of the deprecated `debug` and `transaction_debug_mode` config options are taken into account.
```

The `log_level` option is not set by default, but determined in the extension to be "info" if nothing is set.

Reply:
Good point. Changed the wording. 👍